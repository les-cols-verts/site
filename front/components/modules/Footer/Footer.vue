<template>
  <footer class="footer">
    <p>Retrouvez les Cols verts sur les r√©seaux sociaux :</p>
    <span class="links">
      <a href="https://www.facebook.com/Les-Cols-verts-136135993614214">
        <img
          class="img"
          src="../../../static/facebook.png"
          alt="Logo Facebook"
        />
      </a>
      <a href="https://twitter.com/ColsVerts">
        <img class="img" src="../../../static/twitter.png" alt="Logo Twitter" />
      </a>
      <a href="https://www.youtube.com/channel/UCdXNACLTJoefPVLMQiOYGKQ">
        <img class="img" src="../../../static/youtube.png" alt="Logo Youtube" />
      </a>
      <a href="https://www.linkedin.com/company/les-cols-verts">
        <img
          class="img"
          src="../../../static/linkedin.png"
          alt="Logo LinkedIn"
        />
      </a>
      <a href="https://www.instagram.com/lescolsverts/">
        <img
          class="img"
          src="../../../static/instagram.png"
          alt="Logo Instagram"
        />
      </a>
    </span>
  </footer>
</template>

<script>
export default {};
</script>

<style>
.footer {
  padding: 1em;
  background-color: #fff;
}

.links {
  display: flex;
  justify-content: space-evenly;
  max-width: 600px;
  margin: auto;
  padding-top: 10px;
}

.img {
  width: 3em;
}
</style>
