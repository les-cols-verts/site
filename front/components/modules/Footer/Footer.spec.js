import Footer from '@/components/modules/Footer/Footer';

describe('Footer', () => {
  it('should match snapshot', () => {
    // When
    const wrapper = shallowMount(Footer);

    // Then
    expect(wrapper).toMatchSnapshot();
  });
});
