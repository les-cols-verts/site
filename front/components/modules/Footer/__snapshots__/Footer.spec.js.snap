// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<footer class="footer">
  <p>Retrouvez les Cols verts sur les r√©seaux sociaux :</p> <span class="links"><a href="https://www.facebook.com/Les-Cols-verts-136135993614214"><img src="../../../static/facebook.png" alt="Logo Facebook" class="img"></a> <a href="https://twitter.com/ColsVerts"><img src="../../../static/twitter.png" alt="Logo Twitter" class="img"></a> <a href="https://www.youtube.com/channel/UCdXNACLTJoefPVLMQiOYGKQ"><img src="../../../static/youtube.png" alt="Logo Youtube" class="img"></a> <a href="https://www.linkedin.com/company/les-cols-verts"><img src="../../../static/linkedin.png" alt="Logo LinkedIn" class="img"></a> <a href="https://www.instagram.com/lescolsverts/"><img src="../../../static/instagram.png" alt="Logo Instagram" class="img"></a></span>
</footer>
`;
